### DISCUSSION TITLE
Frigate notification blueprint with nabucasa url access

### DISCUSSION THREAD

**Discussion by PskNorz** [blakeblackshear/frigate#13217] [Ask A Question]
### What is your question:

Hello again,

I am trying to set up the notification blueprint.

My Problem is when i am using blank or nabucasa link for external access on the base url option, when i try to use for the attachment “Snapshot with boudingbox” or Snapshot cropped or Snapshot cropped with bounding box option i get 404 errors.

The author of the blueprint told that this is not a fault of the blueprint.

![IMG_9968](https://github.com/user-attachments/assets/b7bfcd75-4685-47e7-adc8-60a61987fe39)

![IMG_9967](https://github.com/user-attachments/assets/5ba12d0a-0022-45af-b0e1-fe5ba96feadf)

**Comment by f1d094** [comment]
If you are using SgtBatten's blueprint, he has a PR for 0.14 that isn't completed yet. Under 0.14 there are now /reviews and /events. The notification blueprint is setup to use /events and need to be updated to use /reviews instead. Using the old blueprint I was receiving 4-5 empty alerts per hour. I don't understand what changed but I wound up rolling back to 0.13.2 as a result. 

PR discussion is here: https://github.com/SgtBatten/HA_blueprints/pull/216
and
Discussion about phantom events and the /events vs /reviews is here: https://github.com/blakeblackshear/frigate/discussions/11847

Hope that helps

**Reply by PskNorz** [reply]
Thanks for the reply.

If i will use the local ip, the notification will work without problem.

The problem is when i am using the nabucasa url or the same if i let it blank.

And yes i am using SgtBatten blueprint.

**Reply by f1d094** [reply]
Ah...I'm afraid I don't use cloud anything and have no knowledge on that front.
